packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -58,13 +58,15 @@ def get_running_agents():
     cluster.post('/mesos/machine/down', json=down_hosts).raise_for_status()
 
     # Wait for replacements
-    test_util.helpers.wait_for_len(get_running_agents, len(cluster.all_slaves), 600)
+    test_util.helpers.wait_for_len(partial(get_running_agents, 'SlaveServerGroup'), len(cluster.slaves), 600)
+    test_util.helpers.wait_for_len(
+        partial(get_running_agents, 'PublicSlaveServerGroup'), len(cluster.public_slaves), 600)
 
     # Reset the cluster to have the replacement agents
-    cluster.slaves = sorted([agent.private_ip for agent in
-                             dcos_launchpad.get_private_agent_ips(state_list=['running'])])
-    cluster.public_slaves = sorted([agent.private_ip for agent in
-                                    dcos_launchpad.get_public_agent_ips(state_list=['running'])])
+    cluster.slaves = sorted([agent.private_ip_address for agent in
+                             get_running_agents('SlaveServerGroup')])
+    cluster.public_slaves = sorted([agent.private_ip_address for agent in
```

Comment:
Can't call those methods because they currently return Host NamedTuples and this needs plain private IP strings. I don't mind making everything use Host tuples instead, but that should be in a separate PR as it will be many changes to touch everywhere cluster.slaves and what not is called (but well worth it)